docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_Connect_a_pooled-connection-factory_to_a_Remote_Artemis_Server.adoc

```diff
@@ -32,6 +32,15 @@ created at step (2).
 /subsystem=messaging-activemq/pooled-connection-factory=remote-artemis:add(connectors=[remote-artemis], entries=[java:/jms/remoteCF])
 ----
 
+In Artemis 1.x topics and queues used had a prefix(jms.topic. and jms.queue.) that were prepended to the destiantion name.
+In Artemis 2.x this is no longer the case, but for compatibility reasons WildFly still prepend those prefixes and tells Artemis to run in compatibility mode.
```

Comment:
s/prepend/prepends

should this be "tells any embedded Artemis server to run in compatibility mode"?  So WF deals with embedded but with remote Artemis you may need this setting.